example/all.yaml

```diff
@@ -71,6 +71,7 @@ linter:
     - package_prefixed_library_names
     - parameter_assignments
     - prefer_adjacent_string_concatenation
+    - prefer_async_when_returning_futures
```

Comment:
I think this needs to get scooted down after `prefer_asserts_in_initializer_lists`.  Integration tests should pass then!